docs/disaster-recovery.md

```diff
@@ -125,6 +151,8 @@ env is the target environment e.g. production
 
 The following variables need to be set: DB_INSTANCE_GUID (the output of the 'Get affected postgres instance guid' step, SNAPSHOT_TIME ("2021-09-14 16:00:00" IMPORTANT - this is UTC time!), passcode (a passcode from [GOV.UK PaaS one-time passcode](https://login.london.cloud.service.gov.uk/passcode)), CONFIRM_PRODUCTION (true) and tag (the docker tag for the application image).
 
+When running the restore-postgres target against a review environment, the `postgres_params` variable in `modules/paas/variables.tf` needs to be temporarily changed to `postgres_params = merge(local.postgres_backup_restore_params, local.postgres_extensions)`.
```

Comment:
I'm not too keen on this line, but it was a quick way to get the `restore-postgres` to work on the review environment. Without changing this an error would appear saying we couldn't use `restore_from_latest_snapshot_of` and `restore_from_point_in_time_of` at the same time. `restore_from_latest_snapshot_of` is only set for when the `app_environment` = review.